### PR TITLE
[WIP]: Bump bdk-android dependency to 0.29.0 in app-advanced-feature

### PR DESCRIPTION
**Pull request by Prakhar-Agarwal-byte**

### CODE REVIEW

File: app-advanced-features/app/src/main/java/com/goldenraven/devkitwallet/data/Wallet.kt [hunk 1]
```diff
@@ -89,60 +84,45 @@ object Wallet {
...
-
-        // technique 2 for adding a list of recipients to the TxBuilder
-        var txBuilder = recipientList.fold(TxBuilder()) { builder, recipient ->
-            builder.addRecipient(recipient.address, recipient.amount)
-        }
-        if (enableRBF) {
-            txBuilder = txBuilder.enableRbf()
-        }
-        return txBuilder.feeRate(feeRate).finish(wallet)
+    fun createTransaction(recipientList: MutableList<Recipient>, amount: ULong, fee_rate: Float): TxBuilderResult {
```
Comment (thunderbiscuit) [inline app-advanced-features/app/src/main/java/com/goldenraven/devkitwallet/data/Wallet.kt:108]: I think you've copied the code from the simple wallet but notice that one of the features of the advanced wallet is the ability to send to multiple recipients (that's why the recipient argument is a list).

**Review by thunderbiscuit — COMMENTED** [review]
Looks good! Few more things to do and we should be good to merge.

**Review by thunderbiscuit — APPROVED** [review]
ACK d72490e8e96ad00fd0489b141f1c32e36d92eddb. Thanks a bunch! Big changes.

Do you mind testing on your emulator one of those "multiple recipients" transaction? Send me a txid link from mempool.space once you have. I think this is ready to merge.

**Comment by Prakhar-Agarwal-byte** [comment]
Transaction ID-   6f4e92a37dab5e0650d8a33a795ef9d6603d3b142320d0ec739b04a9f141ab00
Mempool link-   https://mempool.space/testnet/tx/6f4e92a37dab5e0650d8a33a795ef9d6603d3b142320d0ec739b04a9f141ab00

**Pull request merged**